src/vmm/src/rpc_interface.rs

```diff
@@ -512,6 +499,12 @@ impl<'a> PrebootApiController<'a> {
             .map_err(VmmActionError::VsockConfig)
     }
 
+    fn set_entropy_device(&mut self, cfg: EntropyDeviceConfig) -> ActionResult {
+        self.boot_path = true;
+        self.vm_resources.build_entropy_device(cfg)?;
+        Ok(VmmData::Empty)
+    }
```

Comment:
Very minor nit:
```suggestion
    fn set_entropy_device(&mut self, cfg: EntropyDeviceConfig) -> ActionResult {
        self.boot_path = true;
        self.vm_resources.build_entropy_device(cfg).map(|_| VmmData::Empty)
    }
```

Reply:
I don't think that works, `build_entropy_device` does not return the same Error type, so it doesn't know how to handle it without the `?` thingy. Am I missing something?